clients/feeder/feeder.go

```diff
@@ -232,6 +235,10 @@ func (c *Client) get(ctx context.Context, queryURL string) (io.ReadCloser, error
 	var res *http.Response
 	var err error
 	wait := time.Duration(0)
+	if c.timeouts == nil {
+		c.timeouts = generateTimeoutsListFromInitial(defaultTimeout, c.maxRetries+1)
```

Comment:
I don't understand why `maxRetries` is part of the constructor? How is maxRetries related with the timeouts array?

Reply:
Yes agree, as mentioned in the comment above, I thought it was a good idea to have a different value for every potential fail.